### PR TITLE
Bug 1999719: persist last viewed tab on topology side panel

### PR DESCRIPTION
**Pull request by sahil143**
**Fixes**: https://issues.redhat.com/browse/ODC-6240

**Analysis / Root cause**: last viewed tab from redux store was not being used while opening the side panel

**Solution Description**: 

- If `selectTab` param is present in the URL then prioritize showing that tab as active tab on side panel
- If `selectTab` param is not present then show the last viewed tab from the redux store
- If query param is not present and redux store value is not present show `Details` tab as the active tab

**Screen shots / Gifs for design review**: 

Kapture 2021-08-31 at 18 42 29

**Unit test coverage report**: 

**Test setup:**

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/helm-plugin/console-extensions.json
Comment (rottencandy) [inline frontend/packages/helm-plugin/console-extensions.json:212]: Needs updating i18n files.

**Comment by invincibleJai** [comment]
@sahil143 take a look at the attached gif, looks like some issue

Kapture 2021-08-31 at 21 58 07

**Comment by sahil143** [comment]
@invincibleJai that scenario is because you have `selectTab` query param in the URL that's why the `Release Notes` tab is selected instead of last viewed.

**Comment by invincibleJai** [comment]
> @invincibleJai that scenario is because you have `selectTab` query param in the URL that's why the `Release Notes` tab is selected instead of last viewed. This is reproducible on the current master as well where queryParam is not cleared.

Thanks , it make sense now and if needed we track this later

**Comment by invincibleJai** [comment]
/approve

cc @rottencandy

File: frontend/packages/dev-console/integration-tests/support/constants/staticText/topology-text.ts
Comment (rohitkrai03) [inline frontend/packages/dev-console/integration-tests/support/constants/staticText/topology-text.ts:4]: I think we don't need this change. We follow this casing convention everywhere in console.

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

File: frontend/packages/dev-console/integration-tests/support/constants/staticText/topology-text.ts
Comment (sahil143) [reply frontend/packages/dev-console/integration-tests/support/constants/staticText/topology-text.ts:4]: @rohitkrai03 some of the cypress tests were failing because of this text because on the Side Bar tabs we have `'Release Notes'`.

Comment (invincibleJai) [reply frontend/packages/dev-console/integration-tests/support/constants/staticText/topology-text.ts:4]: I think he means in tab as well we should show `Release notes` as text

Comment (rohitkrai03) [reply frontend/packages/dev-console/integration-tests/support/constants/staticText/topology-text.ts:4]: Yeah, I meant the tab was originally named `Release notes`, right?

**Comment by rohitkrai03** [comment]
@sahil143 You'll need to rebase the PR. My PR that changes a lot of helm e2e tests got merged. It should fix the test issues you were seeing as well.

**Comment by rohitkrai03** [comment]
@makambalaji The tests are still failing here for helm even after my PR got merged. Now it's failing due to some accessibility violations. Can you take a look?

**Review by rohitkrai03 — APPROVED** [review]
/lgtm

**Comment by makambalaji** [comment]
There is one accessibility issue in helm chart in before hook method. so fixed it in #9727

/hold

**Comment by makambalaji** [comment]
/unhold
PR: #9727 is in merge pool

**Pull request merged**
